src/dmd/globals.d

```diff
@@ -225,26 +225,30 @@ extern (C++) struct Param
     Array!(const(char)*) modFileAliasStrings; // array of char*'s of -I module filename alias strings
     Array!(const(char)*)* imppath;      // array of char*'s of where to look for import modules
     Array!(const(char)*)* fileImppath;  // array of char*'s of where to look for file import modules
-    const(char)[] objdir;                // .obj/.lib file output directory
-    const(char)[] objname;               // .obj file output name
-    const(char)[] libname;               // .lib file output name
+    const(char)[] objdir;               // .obj/.lib file output directory
+    const(char)[] objname;              // .obj file output name
+    const(char)[] libname;              // .lib file output name
 
     bool doDocComments;                 // process embedded documentation comments
     const(char)[] docdir;               // write documentation file to docdir directory
     const(char)[] docname;              // write documentation file to docname
     Array!(const(char)*) ddocfiles;     // macro include files for Ddoc
 
     bool doHdrGeneration;               // process embedded documentation comments
-    const(char)[] hdrdir;                // write 'header' file to docdir directory
-    const(char)[] hdrname;               // write 'header' file to docname
+    const(char)[] hdrdir;               // write 'header' file to docdir directory
+    const(char)[] hdrname;              // write 'header' file to docname
```

Comment:
I think its about time to make this a struct.

Reply:
Probably out of scope for this, though?

Reply:
For this PR yes, thats why I suggested to do it in a separate PR.

Reply:
Oh, I see; I thought you were referring to the comment alignment when you said that